deploy/Dockerrun.aws.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
was this file created via the make CLI command or did you simply update the values? if it is created without make it needs to be redone as this is also the output from us uploading the docker image.